test/dist_autograd_test.py

```diff
@@ -1228,6 +1255,40 @@ def test_backward_simple_python_udf(self):
                 loss = ret.sum()
                 local_grads = self._verify_backwards(exec_mode, [loss], context_id, local_grads, t1, t2)
 
+    @dist_init
+    def test_backward_simple_script_call(self):
+        # Run the same code locally and with dist autograd and verify gradients
+        # are same.
+        local_grads = None
+        t1 = torch.rand((3, 3), requires_grad=True)
+        t2 = torch.rand((3, 3), requires_grad=True)
+        for exec_mode in [ExecMode.LOCAL, ExecMode.RPC_SYNC_SCRIPT_CALL, ExecMode.RPC_ASYNC_SCRIPT_CALL]:
+            with dist_autograd.context() as context_id:
+                if ExecMode.LOCAL == exec_mode:
+                    ret = self._exec_func(exec_mode, my_script_add, t1, t2)
+                else:
+                    ret = self._exec_func(exec_mode, _qualified_name(my_script_add), t1, t2)
```

Comment:
Looks like the user needs to call `_qualified_name(func)` right now? Can we make it such that you can just pass `func` into the API and we call `_qualified_name(func)` internally?